### DISCUSSION TITLE
Get all markers from LottieAnimationView

### DISCUSSION THREAD

**Discussion by ArtMikhailov** [airbnb/lottie-ios#2166] [General]
It would be very useful in some cases to get all markers available as array of Marker entities with all properties. Something like:

```swift
public struct Marker {
    public let cm: String
    public let tm: Float
    public let dr: Float
}

let markers: Array<Marker> = animationView.getMarkers()
```

**Comment by calda** [comment]
There is a public `markerNames` property on `LottieAnimation`: https://github.com/airbnb/lottie-ios/blob/master/Sources/Public/Animation/LottieAnimation.swift#L111

If there is other information about markers that you'd like access to, could you share more details about your use case? Getting the list of marker names should be sufficient for most use cases, since `LottieAnimationView` lets you play markers by name.
